sharding/data_modeling.rst

```diff
@@ -24,12 +24,77 @@ If your situation resembles either of these cases then the next step is to decid
 Distributing by Tenant ID
 =========================
 
-The multi-tenant architecture uses a form of hierarchical database modeling to distribute queries across nodes in the distributed cluster. The top of the data hierarchy is known as the *tenant id*, and needs to be stored in a column on each table. Citus inspects queres to see which tenant id they involve and routes the query to a single physical node for processing, specifically the node which holds the data shard associated with the tenant id. Running a query with all relevant data placed on the same node is called :ref:`colocation`.
```

Comment:
@begriffs -- I might have forgotten our motivation for making this change. The paragraph that we removed looks pretty good to me. Why did we decide to remove them?